### ISSUE TITLE
/proxy is duplicating the url path

### ISSUE THREAD

**Issue by jgallen23** [phonegap/connect-phonegap#156]
For some reason when I'm running `phonegap serve` the proxied url is coming through twice:

Here's the app request:
```
  160511/15:02:51: app.example.com 108.0.16.44 - - [11/May/2016:22:02:51 +0000] "GET /proxy/http%3A%2F%2Fapi.example.com/api HTTP/1.1" 401 104 "-" "Mozilla/5.0 (X11; CrOS x86_64 8172.25.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.37 Safari/537.36"
```

Here's the request on the api server side:
```
 160511/15:02:51: api.example.com 172.17.0.1 - - [11/May/2016:22:02:51 +0000] "GET /api/api HTTP/1.1" 401 104 "-" "Mozilla/5.0 (X11; CrOS x86_64 8172.25.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.37 Safari/537.36"
```

notice the `/api/api`.  I've tried many different combinations and they are always duplicated.  I'm using the latest version of phonegap cli from npm.

**Comment by surajpindoria** [comment]
Hi @jgallen23,

Can you try this using version 6.2.0, `npm install -g phonegap@6.2.0`, and tell me if you get the same results.

**Comment by jgallen23** [comment]
that fixed the issue.

**Comment by Muffinman** [comment]
I'm still getting this on 6.2.2?

    http://localhost:3000/proxy/https%3A%2F%2Fwww.*******.co.uk%2Fpages%2F1

Comes through in the logs as

     62.232.*.* - - [12/May/2016:15:39:44 +0100] "GET /pages/1/pages/1 HTTP/1.1" 200 787 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.94 Safari/537.36"

**Comment by surajpindoria** [comment]
This has been fixed in the latest version, 6.2.3.
